Packs/DatadogCloudSIEM/Integrations/DatadogCloudSIEM/DatadogCloudSIEM.yml

```diff
@@ -0,0 +1,1470 @@
+      description: A flag indicating whether the incident caused customer impact.
+      type: Boolean
+    - contextPath: Datadog.Incident.attributes.notification_handles.display_name
+      description: The name of the notified handle.
+      type: String
+    - contextPath: Datadog.Incident.attributes.notification_handles.handle
+      description: The email address used for the notification.
+      type: String
+    - contextPath: Datadog.Incident.attributes.created
+      description: Timestamp when the incident was created.
+      type: String
+    - contextPath: Datadog.Incident.attributes.modified
+      description: Timestamp when the incident was last modified.
+      type: String
+    - contextPath: Datadog.Incident.attributes.detected
+      description: Timestamp when the incident was detected.
+      type: String
+    - contextPath: Datadog.Incident.attributes.customer_impact_duration
+      description: Length of the incident's customer impact in seconds. Equals the difference between customer_impact_start and customer_impact_end.
+      type: Number
+    - contextPath: Datadog.Incident.attributes.time_to_detect
+      description: The amount of time in seconds to detect the incident. Equals the difference between customer_impact_start and detected.
+      type: Number
+    - contextPath: Datadog.Incident.attributes.time_to_repair
+      description: The amount of time in seconds to resolve customer impact after detecting the issue. Equals the difference between customer_impact_end and detected.
+      type: Number
+    - contextPath: Datadog.Incident.attributes.time_to_internal_response
+      description: The amount of time in seconds to call incident after detection. Equals the difference of detected and created.
+      type: Number
+    - contextPath: Datadog.Incident.attributes.time_to_resolve
+      description: The amount of time in seconds to resolve the incident after it was created. Equals the difference between created and resolved.
+      type: Number
+    - contextPath: Datadog.Incident.attributes.fields.severity.value
+      description: The severity of the incident.
+      type: String
+    - contextPath: Datadog.Incident.attributes.fields.state.value
+      description: The status of the incident.
+      type: String
+    - contextPath: Datadog.Incident.attributes.fields.detection_method.value
+      description: Specify how the incident was detected with these default options - customer, employee, monitor, other, or unknown.
+      type: String
+    - contextPath: Datadog.Incident.attributes.fields.root_cause.value
+      description: This text field allows you to enter the description of the root cause, triggers, and contributing factors of the incident.
+      type: String
+    - contextPath: Datadog.Incident.attributes.fields.summary.value
+      description: Summary of incident.
+      type: String
+    - contextPath: Datadog.Incident.relationships.created_by_user.data.id
```

Comment:
The descriptions for the following are all the same.
....created_by_user.data.id
....modified_by_user.data.id
...last_modified_by_user.data.id
....commander_user.data.id
Is that correct?